### ISSUE TITLE
Support hunspell 1.7

### ISSUE THREAD

**Issue by dorner** [postmodern/ffi-hunspell#25]
The newest version of hunspell is 1.7.0. If this is installed, ffi-hunspell won't work: 

```
dyld: lazy symbol binding failed: Symbol not found: __ZN8HunspellC2EPKcS1_S1_
```

**Comment by scepticulous** [comment]
It seems as if master no includes checking for libhunspell 1.7, but there is no new gem version released yet.
Would that be possible? I am using ffi-hunspell in a gem and therefore can't just point to a git branch. Would love a 0.4.x being released for that :bowing_man:

**Comment by postmodern** [comment]
ffi-hunspell 0.5.0 has been released with 1.7 support. Sorry for the long delay.
